packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactions/CoinmarketTransaction/CoinmarketTransactionInfo.tsx

```diff
@@ -27,8 +27,10 @@ export const CoinmarketTransactionInfo = ({ trade }: CoinmarketTransactionInfoPr
 
     return (
         <Text margin={{ top: spacings.xs }} variant="tertiary" typographyStyle="label" as="div">
-            {tradeType} • <FormattedDate value={date} date time /> •{' '}
-            <CoinmarketTransactionStatus trade={trade} />
+            <Row alignItems="center" flexWrap="wrap">
+                {tradeType} • <FormattedDate value={date} date time /> •{' '}
```

Comment:
Please use new `InfoPair` component here 🙏

See here: https://dev.suite.sldev.cz/components/develop/?path=/story/infopair--info-pair

Reply:
It would be lovely to have it dynamic, not only to the left and right parts. :) But it is nice anyway.

Reply:
We hear you :D 

Please wait for merging [this](https://github.com/trezor/trezor-suite/pull/15627) (should be soon) and refactor it to `InfoSegments`

Reply:
It's merged

Reply:
Updated